Documentation/migration-scripts/python-scripts/fetching_and_auto_mapping_groups.py

```
@@ -0,0 +1,328 @@
+
+def get_session(url, username, password):
+    res = requests.post(
+        f"{url}/session",
+        json={"username": username, "password": password}
+    )
+    res.raise_for_status()
+    return res.json()["id"]
+
+
+def get_groups(url, session_id):
+    return requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)